.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "extends": "airbnb-base",
+    "plugins": [
+        "import"
+    ]
+};
```

Comment:
It appears to work with ES6. That said, if we just pull in the config ourselves, we can tweak it if we need to.